devops/windows/base-docker-wsl/docker-stack.yml

```diff
@@ -0,0 +1,268 @@
+version: '3.8'
+
+networks:
+  backend:
+    name: backend
+    driver: overlay
+    attachable: true
+  frontend:
+    name: frontend
+    driver: overlay
+    attachable: true
+
+services:
+
+  postgresql:
+    image: skunkworks2:5000/v2/bitnami/postgresql:15.4.0
```

Comment:
and i think image repository should be `${IMAGE_REGISTRY}`

Reply:
same here using
  envsubst < ./conf/stack/docker-stack-${SPEC_SETTINGS}.yml > ./docker-stack.yml